src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -67,6 +67,18 @@ internal bool IsArrayIndex
             }
         }
 
+        private class DeconstructionUncommonData : UncommonData
```

Comment:
nestedConversions might go to the common base though

---
In reply to: [100886606](https://github.com/dotnet/roslyn/pull/17027#discussion_r100886606) [](ancestors = 100886606)